jssource/Minifier.php

```diff
@@ -121,6 +121,12 @@ class Minifier
      */
     static public function minify($js, $options = array())
     {
+        global $sugar_config;
+
+        if(isset($sugar_config['developerMode']) && $sugar_config['developerMode'] === true) {
```

Comment:
I think we should make a developer_mode a prefix.

In the system settings, the developer mode check box should reveal other options including this one. Perhaps your stack trace feature #3486 could also be an option.

I have changed this to developer_mode_disable_minifier.